src/SFA.DAS.AssessorService.ApplyTypes/Roatp/GatewayApplicationSummaryItem.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace SFA.DAS.AssessorService.ApplyTypes.Roatp
+{
+    public class GatewayApplicationSummaryItem
```

Comment:
As commented on the Apply side, can we re-use this across Gateway, FHA and Assessor or do we want to store different information for different reviewer types?

Reply:
Make this generic so we now have RoatpApplicationSummaryItem